xonsh/built_ins.py

```diff
@@ -792,6 +792,7 @@ def run_subproc(cmds, captured=False):
     Lastly, the captured argument affects only the last real command.
     """
     specs = cmds_to_specs(cmds, captured=captured)
+    events.on_command_started.fire(cmd=specs)
```

Comment:
The keyword here should be `specs`, not `cmd`.  Also this isn't really after the command started. It is more like "after commands specified" or "before pipeline execution"